pkg/commands/process/settings/rules/ruby/third_parties/airbrake.yml

```
@@ -0,0 +1,74 @@
+languages:
+  - ruby
+patterns:
+  - pattern: |
+      Airbrake.$<METHOD>($<...>$<DATA_TYPE>$<...>)
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+      - variable: METHOD
+        values:
+          - notify
+          - notify_sync
+          - notify_request
+          - notify_request_sync
+          - notify_query
+          - notify_query_sync
+          - notify_performance_breakdown
+          - notify_performance_breakdown_sync
+          - notify_queue
+          - notify_queue_sync
+  - pattern: |
+      Airbrake.$<METHOD>() do |$<NOTIFY:identifier>|
+        $<NOTIFY>[][] = $<DATA_TYPE>
+      end
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+      - variable: METHOD
+        values:
+          - notify
+          - notify_sync
+          - notify_request
+          - notify_request_sync
+          - notify_query
+          - notify_query_sync
+          - notify_performance_breakdown
+          - notify_performance_breakdown_sync
+          - notify_queue
+          - notify_queue_sync
+  - pattern: |
+      Airbrake.merge_context($<...>$<DATA_TYPE>$<...>)
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+  - pattern: |
+      def to_airbrake
+        $<...>$<DATA_TYPE>$<...>
```

Because it's in a method body, the `...` is implied. You should only need to use the ellipsis in either an arguments list, or after a method call to specify an optional block.

```suggestion
        $<DATA_TYPE>
```